### ISSUE TITLE
Line 469

### ISSUE THREAD

**Issue by CanDKD** [agrc/uic-forms#3]
We need to include more of the Utah UIC gdb InspectionType codes to this line as follows:

InspectionType = new[] {"RP", "OT", "NW", "FI"}

**Comment by steveoh** [comment]
Which line are you wanting to reference here?

**Comment by steveoh** [comment]
https://github.com/agrc/uic-forms/blob/master/uic-forms/Program.cs#L469?

```cs
InputMonadGenerator.CreateMonadFor(new[] {1, 3, 4, 5}, "VB5_{class}", new QueryParams
                    {
                        InspectionType = new[] {"RP"}
                    }, sevenFiveTwenty.GetInspections, ref formInfo);
```

so `RP` to `"RP", "OT", "NW", "FI"`

**Comment by CanDKD** [comment]
469


*Candace C. Cady, PG | Underground Injection Control (UIC) Program*
phone:  801.536.4352

office location:    195 North 1950 West, Salt Lake City, Utah  84116
mailing address:  Utah Division of Water Quality
                          PO Box 144870
                          Salt Lake City, Utah   84114

*Emails to and from this email address may be considered public records and
thus subject to Utah GRAMA requirements.*

**Comment by CanDKD** [comment]
Yes, we will use the 7520-3 form; field VB5_1 through VB5_5 to capture the InspectionType codes in the Utah UIC gdb that are not otherwise captured in VB1 through VB4 (_1 to _5)

**Comment by steveoh** [comment]
fixed in version 1.1.4
